package-lock.json

```diff
@@ -1635,7 +1635,7 @@
     "big.js": {
       "version": "5.2.2",
       "resolved": "https://registry.npmjs.org/big.js/-/big.js-5.2.2.tgz",
-      "integrity": "sha512-vyL2OymJxmarO8gxMr0mhChsO9QGwhynfuu4+MHTAW6czfq9humCB7rKpUjDd9YUiDPU4mzpyupFSvOClAwbmQ==",
+      "integrity": "sha1-ZfCvOC9Xi83HQr2cKB6cstd2gyg=",
```

Comment:
Don't commit this file.

But ... @apb7, this suggests your script is not working? Might want to work with @Jamesjay4199 to figure out why.

Reply:
@Jamesjay4199, I don't see this comment as resolved, was this an expected change?

/cc @seanlip

Reply:
This should have been fixed, but let's leave it for now. I think there's been subsequent package-lock.json updates that have regenerated the file so this is probably moot.

That said, @Jamesjay4199, please make sure to resolve all comments before merge -- and if a merge happens prior to that by accident, immediately do a follow-up PR to resolve any trailing comments. Thanks!

Reply:
Okay.
I think I missed this.
I'll make sure to do this.